modules/serverless-installing-cli-linux-rpm.adoc

```diff
@@ -1,20 +1,38 @@
 // Module is included in the following assemblies:
 //
-// serverless/knative-client.adoc
+// serverless/installing_serverless/installing-kn.adoc
 
 [id="installing-cli-linux-rpm_{context}"]
-= Installing the `kn` CLI for Linux using an RPM
-
-For Red Hat Enterprise Linux (RHEL), you can install `kn` as an RPM if you have an active {product-title} subscription on your Red Hat account.
+= Installing the Knative CLI for Linux using an RPM
 
+For Red Hat Enterprise Linux (RHEL), you can install the `kn` CLI as an RPM if you have an active {product-title} subscription on your Red Hat account.
 
 .Procedure
-* Use the following command to install `kn`:
+
+* Enter the following commands to install the `kn` CLI:
```

Comment:
Not critical, but I'd consider making these separate steps, like is done for the `oc` RPM install procedure: https://docs.openshift.com/container-platform/4.6/cli_reference/openshift_cli/getting-started-cli.html#cli-installing-cli-rpm_cli-developer-commands